keel-web/src/test/resources/examples/cluster-example.yml

```diff
@@ -22,7 +22,7 @@ environments:
     spec:
       health:
         terminationPolicies:
-        - Default
+        - default
```

Comment:
I had to change a bunch of things in these example files to lowercase because JSON schema has no way of specifying an enum as case-insensitive. We'll still actually parse case-insensitively but it's not really practical to list all the possible variations in the schema. Still thinking about ways to improve this but I don't want it to just be free-form text because that's not very helpful for users.

Reply:
It's kinda sucky, but could you enumerate lower case, upper case, and all caps as separate options as a workaround, e.g. (I don't know the actual syntax) `[default, Default, DEFAULT]`?